_includes/prev-conf-programmes-nav.html

```diff
@@ -0,0 +1,24 @@
+<ul>
```

Comment:
You don't need to add this, there is already an include for previous conferences - not quite sure what's going on here